_posts/blog/2018-09-12/2018-09-12-how-to-get-more-buyers-on-openbazaar-with-categories-and-tags.markdown

```diff
@@ -0,0 +1,145 @@
+social_image: OpenBazaar demo store - How To Get More Buyers on OpenBazar with Categories and Tags.png
+---
+
+_by Jason Hotelling, OpenBazaar Front End Developer_
+
+Hello! We hope you're having a great time selling and sharing on the OpenBazaar platform. 
+
+OpenBazaar stores boast the fastest setup time of any e-commerce platform. With very little effort, you can set up your store and add listings in minutes. Then, a little extra work can give buyers an exceptional experience to make them want to visit again and again! 
+
+Today we are going to talk about how to use tags and categories to help users find and browse your store with ease.
+
+### Before We Get Started
+
+If you haven't already, please follow our guides to get started with OpenBazaar. They contain instructions and tutorials that span from _Download_ through _Browsing_ and _Selling_.
+
+[The Beginner's Guide to Buying Goods, Services and Cryptocurrency on Openbazaar](https://openbazaar.org/blog/The-Beginners-Guide-to-Buying-Goods-Services-and-Cryptocurrency-on-OpenBazaar/)
+
+If you are new, we also recommend watching our [OpenBazaar 2.0 Tutorial](https://www.youtube.com/watch?v=VzlerMJMOu8) on YouTube.
+
+{% include modules/embeded-video.html url="https://www.youtube.com/embed/VzlerMJMOu8"%}
+
+### Step 1: Tags Are for Search
+
+**Tags** are what the search engines use to determine which listings to show to a potential buyer so they are very important. Most users will discover your store through a listing that they find in search. For example, if a user is looking for t-shirts, they first navigate to the **Discover** tab in OpenBazaar. Then they will select a search provider and enter a search term such as "t-shirt" or "t shirt". In order for potential buyers to find _your shirts_, you must enter any relevant tags on your listings. 
+
+In this example, we enter tags that may match what users enter in Search in the OpenBazaar Discover view.
+
+![Add tags to your OpenBazaar listing](Add tags to your OpenBazaar listing.gif "Add tags to your OpenBazaar listing")
+
+> Tip: Make sure to add your store name as a tag too so when people search for your store name, your listings will show up.
+
+### Step 2: Categories and Filters in Your Store
+
+Once users click on your listing, they arrive in your store. This is where you have an opportunity to hook them and encourage them to shop around more. Your **Store View** will show potential customers all your available listings. 
+
+![OpenBazaar demo store - How to get more buyers on OpenBazaar with categories and tags](OpenBazaar demo store - How To Get More Buyers on OpenBazar with Categories and Tags.png "OpenBazaar demo store - How to get more buyers on OpenBazaar with categories and tags")
+
+It will make it easier for buyers to shop if you classify your listings. Take time to become familiar with the Store View browsing experience. It will aid you later in listing planning, classification, organization and creation.
```

Comment:
"You should take time to..." otherwise there isn't a subject for the verb.

> It will aid you later in listing planning, classification, organization and creation.

This sounds like there should be a comma after listing, as if it's one of the items in the list.

I suggest "It will aid you later in planning and creating your listings." (classifying was already mentioned in the first sentence, and organizing is probably not needed).